graphql/queryResolvers/addComment.ts

```diff
@@ -0,0 +1,21 @@
+import { prisma } from '../../prisma'
+import { MutationAddCommentArgs } from '../../graphql'
+import { Context } from '../../@types/helpers'
+
+export const addComment = async (
+  _parent: void,
+  arg: MutationAddCommentArgs,
+  _ctx: Context
+) => {
+  const { line, submissionId, authorId, fileName, content } = arg
```

Comment:
Get `authorId` from `req.session.user`, don't send it as a parameter for the query.

Reply:
Fixed.